dependencies/Makefile.mac

```diff
@@ -69,56 +73,64 @@ $(WEBOTS_DEPENDENCY_PATH)/freetype-2.9/objs/.libs/libfreetype.6.dylib:
 
```

Comment:
@fabienrohrer, I have problems understanding how we use `libfreetype` on mac.
The archive is extracted in the `dependencies` folder and nothing is copied in any released folder.
If Webots is linked with `libfreetype.6.dylib` how does it works in the release? And if not, why is this dependency required?

Reply:
Ok I understand now, we are not linking with `libfreetype.6.dylib` as this Makefile may let think, but with `libfreetype.a`,  I will make it clearer in this Makefile.